designs/wahid/src/front.mjs

```diff
@@ -336,6 +336,7 @@ function wahidFront({
   // Title
   macro('title', {
     at: points.flbCpTop,
+    title: 'wahid:front',
```

Comment:
@HaasJona I've resolved the translation issue in #6360.

Could you remove the namespace prefix from the title? 
It's loaded as the default namespace, so it will *just work*. But, more importantly, this allows people to override the translations when extending a pattern, as the default namespace will then be the one of their extended pattern.

I should explain this in the docs too :thinking:

Reply:
I have removed the prefixes